js/js-unit-testing/js-unit-testing_people/index.js

```diff
@@ -2,18 +2,54 @@ export function getFirstNames(people) {
   return people.map((person) => person.firstName);
 }
 
-export function getFullNames(people) {}
+export function getFullNames(people) {
+  return people.map((person) => `${person.firstName} ${person.lastName}`);
+}
+
+export function getNameAndAge(people) {
+  return people.map((person) => `${person.lastName} (${person.age})`);
+}
 
-export function getNameAndAge(people) {}
+export function getPeopleByAge(people, age) {
+  const peopleList = people.filter((person) => person.age >= age);
+  const matches = peopleList.map(
+    (people) => `${people.firstName} ${people.lastName}`
+  );
 
-export function getPeopleByAge(people, age) {}
+  return matches;
+}
 
-export function getPeopleNamesOlderThan(people, age) {}
+export function getPeopleNamesOlderThan(people, age) {
+  const peopleList = people.filter((person) => person.age > age);
 
-export function getPeopleByLastName(people, lastName) {}
+  const olderThan = peopleList.map(
+    (people) => `${people.firstName} ${people.lastName}`
+  );
 
-export function findPersonById(people, id) {}
+  return olderThan;
+}
 
-export function isAnyoneOlderThan(people, age) {}
+export function getPeopleByLastName(people, lastName) {
+  const peopleList = people.filter((person) => person.lastName === lastName);
+  const matches = peopleList.map((people) => `${people.lastName}`);
```

Comment:
I think what is expected here is actually an array of objects out of people again, otherwise the return of the function equals its argument --> that would be a bit disappointing no? ;) So just use the filter return the outcome and leave the mapping out :)